inc/classes/class-rocket-critical-css.php

```diff
@@ -567,7 +567,7 @@ public function async_css( $buffer ) {
 			}
 
 			$preload = str_replace( 'stylesheet', 'preload', $tags_match[1][ $i ] );
-			$onload  = preg_replace( '~' . $tags_match[3][ $i ] . '~iU', ' as="style" onload=""' . $tags_match[3][ $i ] . '>', $tags_match[3][ $i ] );
+			$onload  = preg_replace( '~' . preg_quote( $tags_match[3][ $i ] ) . '~iU', ' as="style" onload=""' . $tags_match[3][ $i ] . '>', $tags_match[3][ $i ] );
```

Comment:
The 2nd argument is missing:
```php
preg_quote( $tags_match[3][ $i ], '~' )
```
Is it normal that you 3 times the same thing?
```php
$a = preg_replace( '~' . $b . '~', 'foo' . $b . 'bar', $b );
```

Reply:
Yes it is, I might need to revisit that, but for now it works